backend/climateconnect_api/views/user_views.py

```diff
@@ -67,12 +67,8 @@ def post(self, request, format=None):
                 user_profile.save()
             return super(LoginView, self).post(request, format=None)
         else:
-            if not User.objects.filter(username=request.data['username']).exists():
-                return Response({
-                    'message': 'Username does not exist. Have you signed up yet?'
-                }, status=status.HTTP_401_UNAUTHORIZED)
             return Response({
-                'message': 'Invalid password.'
+                'message': _('Invalid username or password.')
```

Comment:
Since users don't have a username but only an email, I'd also change that here (even though it was also wrong before). Accordingly in german we could then also use "Ungültige E-Mail oder Passwort"
```suggestion
                'message': _('Invalid email or password')
```

Reply:
I changed it and updated the django language files.